integ-test/src/test/java/org/opensearch/sql/sql/QueryIT.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.sql;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_BEER;
+
+import java.io.IOException;
+import org.json.JSONObject;
+import org.junit.Test;
+import org.opensearch.sql.legacy.SQLIntegTestCase;
+
+public class QueryIT extends SQLIntegTestCase {
+    @Override
+    public void init() throws IOException {
+        loadIndex(Index.BEER);
+    }
+
+    @Test
+    public void all_fields_test() throws IOException {
+        String query = "SELECT * FROM "
+                + TEST_INDEX_BEER + " WHERE query('*:taste')";
```

Comment:
I thought the wildcard syntax doesn't work?  Or does it work with just `*`?

Reply:
I thought so too... I thought we determined that they didn't work when we had our 1:1. I played around with it trying different combinations...

The following all seem to work when passed in as the query for query_string:
```
"* : taste"
"* : ?aste"
"Tags : *aste"
"Tags : Ta?te"
"Tags : Ta*te"
```
I tried these according to [this](https://lucene.apache.org/core/2_9_4/queryparsersyntax.html#Fields:~:text=of%20searching%20options.-,Wildcard%20Searches,-Lucene%20supports%20single) documentation for Lucene, but it doesn't seem to follow it exactly since it also allows `*` and `?` to be the first character of a search. Should we add tests for all of these cases as well? I can't find a specific Doc to reference for what should/shouldn't be supported in the query so I'm not really sure what other behaviour we should be testing for that we might be missing.

Reply:
Added Unit Tests in f2d77c634b1869229cbb3aa2f6952403d26fe30c